repose-aggregator/commons/commons-utilities/src/test/java/org/openrepose/commons/utils/servlet/http/RouteDestinationTest.java

```diff
@@ -21,58 +21,55 @@
 
 import org.junit.Before;
 import org.junit.Test;
-import org.junit.experimental.runners.Enclosed;
-import org.junit.runner.RunWith;
 
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.lessThan;
+import static org.hamcrest.Matchers.not;
 import static org.junit.Assert.*;
 
 /**
  * @author malconis
  */
-@RunWith(Enclosed.class)
 public class RouteDestinationTest {
 
-    public static class WhenCreatingDestinations {
+    private RouteDestination routeDst1, routeDst2, routeDst3, routeDst4;
 
-        private RouteDestination routeDst1, routeDst2, routeDst3, routeDst4;
+    @Before
+    public void setUp() {
 
-        @Before
-        public void setUp() {
+        routeDst1 = new RouteDestination("dst1", "/service/dst1", new Float(1.0));
+        routeDst2 = new RouteDestination("dst2", "/service/dst2", new Float(-1.0));
+        routeDst3 = new RouteDestination("dst3", "/service/dst3", new Float(1.0));
+        routeDst4 = new RouteDestination("dst1", "/service/dst4", new Float(1.0));
 
-            routeDst1 = new RouteDestination("dst1", "/service/dst1", new Float(1.0));
-            routeDst2 = new RouteDestination("dst2", "/service/dst2", new Float(-1.0));
-            routeDst3 = new RouteDestination("dst3", "/service/dst3", new Float(1.0));
-            routeDst4 = new RouteDestination("dst1", "/service/dst4", new Float(1.0));
-
-        }
+    }
 
-        @Test
-        public void shouldReturnDestinationWithHighestQuality() {
+    @Test
+    public void shouldReturnDestinationWithHighestQuality() {
 
-            int compared = routeDst1.compareTo(routeDst2);
+        int compared = routeDst1.compareTo(routeDst2);
 
-            assertEquals(compared, 1);
-        }
+        assertEquals(compared, 1);
```

Comment:
In general i do prefer hamcrest matchers, but dealing with the those assertions was way outside the scope of my story.